### ISSUE TITLE
How about use golang as the backend?

### ISSUE THREAD

**Issue by huapox** [jc21/docker-registry-ui#19]
First this is awesome project, with lite useful and refreshing frontend.

And now it's with `node` for the backtend, as we can see the image's size at `165.72 MB` in dockerhub alter compressed. and the runtime mem up to about `40Mb`.

There is just about 3 apis between backtend and frontend, so come with a idea of changing the backtend to `golang` of implement. for lite image size(about `40Mb`) and lower run memory(about `8Mb`).

**Comment by jc21** [comment]
Yeah I am trying to switch to a golang backend for another one of my projects and it's so much nicer. I really haven't devoted any time to this registry ui in almost a couple of years though.

**Comment by huapox** [comment]
> Yeah I am trying to switch to a golang backend for another one of my projects and it's so much nicer. I really haven't devoted any time to this registry ui in almost a couple of years though.

Great! and if with your permission I'll devote some time to this work.  thx~

**Comment by tomaswarynyca** [comment]
Any news about migration?
[Fiber](https://gofiber.io/) could be used as backend
